modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/bnd.bnd

```diff
@@ -17,7 +17,7 @@ Web-ContextPath: /dynamic-data-${artifactId}-form-field
 -metatype: *
 -plugin.jsp: com.liferay.ant.bnd.jsp.JspAnalyzerPlugin
 -plugin.metatype: com.liferay.ant.bnd.metatype.MetatypePlugin
-#if (${liferayVersion.startsWith("7.1")} || ${liferayVersion.startsWith("7.2")} || ${liferayVersion.startsWith("7.3")})
+#if (${liferayVersion.startsWith("7.1")})
```

Comment:
@lawrence-lee, why don't we need it for versions from 7.2 to 7.4 anymore?

Reply:
We cant support the maven case from 7.2-7.4 because we dont have an equivalent `jsCompile` configuration in maven like we do in gradle.